src/pages/Lessons.vue

```diff
@@ -171,22 +171,53 @@ export default {
       this.isCreateModalOpen = true;
     },
     deleteLessons() {
-      this.selectedIds.forEach(item => this.deleteLesson(item.id));
+      const promises = this.selectedIds.map(item => this.deleteLesson(item.id));
+
+      const pending = this.showNotif('pendingMessage', 'Удаление...');
+
+      Promise.all(promises)
+        .then(() => {
+          pending();
```

Comment:
for pending use Promise.prototype.finally()

Reply:
Done.